codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2GoAwayFrame.java

```diff
@@ -87,6 +93,17 @@ public Http2GoAwayFrame setExtraStreamIds(int extraStreamIds) {
     }
 
     @Override
+    public Http2GoAwayFrame setLastStreamId(int lastStreamId) {
+        this.lastStreamId = lastStreamId;
+        return this;
+    }
+
+    @Override
+    public Integer lastStreamId() {
+        return lastStreamId;
```

Comment:
I think I was going to go with something more like a List<Http2Stream>. That way there wouldn't be any need for a second `int → stream` mapping in the multiplexing handler. It isn't without problems, but seemed simpler than keeping another map (as a map may require every handler between the multiplexing codec and the frame codec to maintain its own map).